### PR TITLE
Add `hasHeadCss` option

### PR DESCRIPTION
**Pull request by ironsidevsquincy**
To read the head css file and store in `res.locals.headCss`

Addresses https://github.com/Financial-Times/next-front-page/pull/758#discussion-diff-62103412 - @matthew-andrews

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -28,6 +30,7 @@ module.exports = function(options) {
 		withAnonMiddleware: false,
 		withBackendAuthentication: false,
 		withRequestTracing: false,
+		hasHeadCss: false,
```
Comment (matthew-andrews) [inline main.js]: i assume that we are currently requiring a one-to-one mapping between apps and head css's.  ie. an app can't have more than one compiled head css file?  (e.g. different routes)

(this is ok, keeps things simple, just checking)

**Comment by matthew-andrews** [comment]
+1

File: main.js (hunk 1 above)
Comment (ironsidevsquincy) [inline main.js]: Correct, though good point, might be something we need in the future. Though if an app has wildly different heads between pages, maybe it shouldn't be within a single app...

Comment (matthew-andrews) [inline main.js]: yeah i agree with that.  quite nice to have an incentive to split things out for a change :]

**Pull request merged**